transport_analysis/velocityautocorr.py

```diff
@@ -262,3 +266,78 @@ def plot_vacf(self, start=0, stop=0, step=1):
             self.times[start:stop:step],
             self.results.timeseries[start:stop:step],
         )
+
+    def sd_gk(self, start=0, stop=0, step=1):
```

Comment:
Another nitpick, but generally I'm not in favor of suber abbeviated naming. `sd_gk` means something to you now, as a developer, but it probably won't mean much to a new user. Maybe `self_diffusivity_gk` or `self_diffusivity_green_kubo` or `diffusivity_green_kubo`?

Reply:
Good point, I'll go with `self_diffusivity_gk` then.